ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/MigrationConfig.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.tools.migrations;
+
+import io.confluent.ksql.properties.PropertiesUtil;
+import io.confluent.ksql.rest.client.KsqlRestClient;
+import io.confluent.ksql.rest.client.KsqlRestClientException;
+import io.confluent.ksql.rest.client.RestResponse;
+import io.confluent.ksql.rest.entity.ServerInfo;
+import io.confluent.ksql.util.KsqlException;
+import java.io.File;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigDef.Importance;
+import org.apache.kafka.common.config.ConfigDef.Type;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MigrationConfig extends AbstractConfig {
+
+  public static final String KSQL_SERVER_URL = "ksql.server.url";
+
+  public static final String KSQL_MIGRATIONS_STREAM_NAME = "ksql.migrations.stream.name";
+  public static final String KSQL_MIGRATIONS_STREAM_NAME_DEFAULT = "migration_events";
+  public static final String KSQL_MIGRATIONS_TABLE_NAME = "ksql.migrations.table.name";
+  public static final String KSQL_MIGRATIONS_TABLE_NAME_DEFAULT = "schema_version";
+  public static final String KSQL_MIGRATIONS_STREAM_TOPIC_NAME =
+      "ksql.migrations.stream.topic.name";
+  public static final String KSQL_MIGRATIONS_TABLE_TOPIC_NAME = "ksql.migrations.table.topic.name";
+  public static final String KSQL_MIGRATIONS_TOPIC_REPLICAS = "ksql.migrations.topic.replicas";
+  public static final int KSQL_MIGRATIONS_TOPIC_REPLICAS_DEFAULT = 1;
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(MigrationConfig.class);
```

Comment:
What log4j config is being used with these loggers by default? Do we need to provide one with the tool?

Reply:
Added a log4j config file that defaults to stdout